forms-flow-api/src/formsflow_api/resources/form_process_mapper.py

```diff
@@ -354,31 +354,49 @@ def get(application_id: int):
 
 
 @cors_preflight("POST,OPTIONS")
-@API.route("/form-create", methods=["POST", "OPTIONS"])
+@API.route("/formio", methods=["POST", "OPTIONS"])
 class FormioFormResource(Resource):
     """Resource for formio form creation."""
 
     @staticmethod
     @auth.require
+    @auth.has_one_of_roles([DESIGNER_GROUP])
     @profiletime
     def post():
         """Formio form creation method."""
         try:
             data = request.get_json()
-            if auth.has_role([DESIGNER_GROUP]):
-                formio_service = FormioService()
-                form_io_token = formio_service.get_formio_access_token()
-                response, status = (
-                    formio_service.create_form(data, form_io_token),
-                    HTTPStatus.CREATED,
-                )
-            else:
-                response, status = (
-                    {
-                        "message": "Permission Denied",
-                    },
-                    HTTPStatus.FORBIDDEN,
-                )
+            formio_service = FormioService()
+            form_io_token = formio_service.get_formio_access_token()
+            response, status = (
+                formio_service.create_form(data, form_io_token),
+                HTTPStatus.CREATED,
+            )
+            return response, status
+        except BusinessException as err:
+            current_app.logger.warning(err.error)
+            return err.error, err.status_code
+
+
+@cors_preflight("PUT,OPTIONS")
+@API.route("/formio/<string:form_id>", methods=["PUT", "OPTIONS"])
+class FormioFormUpdateResource(Resource):
+    """Resource for formio form Update."""
+
+    @staticmethod
+    @auth.require
```

Comment:
You can remove @auth.require as the other decorator would take care of this check.